build.gradle

```diff
@@ -218,4 +190,13 @@ clover {
   }
 }
 
-compileJava.dependsOn setVersion
+jar.dependsOn gitRevList
```

Comment:
Any reason why this shouldn't be compileJava?

Someone using ./gradlew run will no longer have revision set appropriately.

Reply:
The revision is no longer embedded in the java code with this change, so adding it as a dependency there isn't necessary. It's only bundled into the jar now. But yeah it means running the code without a jar means the revision won't be available.